README.md

```diff
@@ -82,13 +82,34 @@ or you can visit to this link for the latest version of dependency: https://sear
     DriverManager.getConnection(yburl);
     ```
 
-  For specifying topology keys you need to set the additional property with a valid comma separated value, for example _topology-keys=region1.zone1,region1.zone2_.
+  For specifying topology keys you need to set the additional property with a valid comma separated value.
 
     ```
-    String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte&load-balance=true&topology-keys=region1.zone1,region1.zone2";
+    String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte&load-balance=true&topology-keys=cloud1.region1.zone1,cloud1.region1.zone2";
     DriverManager.getConnection(yburl);
     ```
 
+### Specifying fallback zones
+
+  You can also specify fallback placements for topology-aware load balancing. Optionally, a placement value can be suffixed with a colon (`:`) followed by a preference value between 1 and 10.
+  A preference value of `:1` means it is a primary placement. A preference value of `:2` means it is the first fallback placement and so on.
+  If no preference value is provided, it is considered to be a primary placement (equivalent to one with `:1`). Example given below.
```

Comment:
So if there are multiple placement zones and no preference is given then what is the order of fallback? Same question as asked earlier.

Reply:
The default preference value is :1. So placements without a preference value are considered as primary placements. This is done to maintain continuity with earlier syntax.